commit a7591ed721b1402703b9d5230923bf05ed6dbea4 (refs/changes/62/3262/2)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-01-29T19:15:54+00:00 (1 year, 8 months ago)
    
    RELENG-612 prefer (git+)https urls
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/.gitmodules b/.gitmodules
index 7f63af3..b3d9976 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,9 +1,9 @@
 [submodule "deps/javascriptlint"]
 	path = deps/javascriptlint
-	url = git://github.com/davepacheco/javascriptlint.git
+	url = https://github.com/davepacheco/javascriptlint.git
 [submodule "deps/jsstyle"]
 	path = deps/jsstyle
-	url = git://github.com/davepacheco/jsstyle.git
+	url = https://github.com/davepacheco/jsstyle.git
 [submodule "deps/sdc-scripts"]
 	path = deps/sdc-scripts
-	url = git@github.com:joyent/sdc-scripts.git
+	url = https://github.com/joyent/sdc-scripts.git
